src/main/java/com/google/codeu/servlets/UserListServlet.java

```diff
@@ -1,22 +1,35 @@
 package com.google.codeu.servlets;
 
+import com.google.codeu.data.Datastore;
+import com.google.gson.Gson;
 import java.io.IOException;
 
+import java.util.Set;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 /**
- * Handles fetching all users for the community page.
```

Comment:
I think "Handles" is more idiomatic; see https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment

> This is a fragment—a noun phrase or verb phrase, not a complete sentence. It does not begin with A {@code Foo} is a..., or This method returns..., nor does it form a complete imperative sentence like Save the record.. However, the fragment is capitalized and punctuated as if it were a complete sentence.

Sorry about potentially misleading you with what I said about complete sentences in another PR review; the fragment/first line of the javadoc should be a sentence fragment.